### PR TITLE
Add github actions for auto-releasing the plugin to Moodle plugins directory

### PR DESCRIPTION
**Pull request by chenli12138**
Task

1. Adding .github/workflows/moodle-release.yml
2. A micro version bump

### CODE REVIEW

**Comment by jwalits** [comment]
Hey @chenli12138 

2 minor notes on this PR

a) Please rename moodle-tool_passwordvalidator.yml to moodle-release.yml to align with how we name the file for other plugins

b) We dont usually put WR numbers for commit messages on Github (as it public facing). Please re-word the commit message to something simple like `Add Moodle release workflow`

**Comment by chenli12138** [comment]
@jwalits  Hi Jwait,

I have renamed the file and also change commit message based on your feedback.

File: .github/workflows/moodle-release.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# .github/workflows/moodle-release.yml
+name: Releasing in the Plugins directory
+
+on:
+  push:
+    branches:
+      - MOODLE_310_STABLE
```
Comment (jwalits) [inline .github/workflows/moodle-release.yml:7]:
```suggestion
      - master
```

Comment (chenli12138) [reply .github/workflows/moodle-release.yml]: Hi @jwalits 

Change has been made for pushing branch

**Pull request merged**